src/status_im/ui/components/copyable_text.cljs

```diff
@@ -81,6 +81,7 @@
 (defn copyable-text-view
   [{:keys [label container-style]} content]
   (let [cue-atom     (reagent/atom false)
+        background-color (when-not (get container-style :background-color) colors/white)
```

Comment:
`(or (get container-style :background-color) colors/white)` ?

Reply:
Yes that's a good idea